doc/manual/src/command-ref/env-common.md

```diff
@@ -44,7 +44,7 @@ Most Nix commands interpret the following environment variables:
     The Nix search path can also be extended using the `-I` option to
     many Nix commands, which takes precedence over `NIX_PATH`.
 
-    If `NIX_PATH` is not set or empty, Nix will look up the following locations for a revision of `nixpkgs`:
+    If `NIX_PATH` environment variable is not set, Nix will fall back to the following list in impure and unrestricted evaluation mode:
```

Comment:
```suggestion
    If `NIX_PATH` is not set at all, Nix will fall back to the following list in impure and unrestricted evaluation mode:
```

No reason to recall it's an environment variable (we have that from context). I suggest to emphasize that being set to empty and not being set is very different.